assets/javascripts/summary/index.js

```diff
@@ -1,9 +1,26 @@
-var map = require('../map.js')
+var ol = require('openlayers')
+var Map = require('../map.js')
 
 function Summary (options) {
   var easting = window.encodeURIComponent(options.easting)
   var northing = window.encodeURIComponent(options.northing)
-  map.loadMap(easting && [easting, northing], true)
+
+  var mapOptions = {
+    point: [parseInt(easting), parseInt(northing)],
+    layers: [
+      new ol.layer.Image({
+        ref: 'fmp',
+        source: new ol.source.ImageWMS({
+          url: '/wms-proxy',
+          serverType: 'geoserver',
+          params: {
+            'LAYERS': 'fmp'
+          }
+        })
+      })
+    ]
+  }
+  Map(mapOptions)
```

Comment:
Yeah that's annoying - I've come across that.

Use either:

this.map = new Map(mapOptions)  // You may want to expose "map" on summary page for testing purposes?

Otherwise, if you don't want to expose it, add quotes - I think standard will be ok with this:

(new Map(mapOptions))